code/modules/mob/living/xenomorphic/life.dm

```diff
@@ -0,0 +1,94 @@
+
+/mob/living/xenomorphic/Life()
```

Comment:
If you're making a new mob/living subtype, without all the legacy of carbons, I strongly suggest that you override the procs called in /mob/living/life.dm instead of creating a monolithic Life() monster.